packages/@lwc/engine-core/src/framework/hooks.ts

```diff
@@ -436,6 +440,55 @@ export function removeElmHook(vnode: VElement) {
     }
 }
 
+// slow path routine
+// NOTE: we should probably more this routine to the synthetic shadow folder
```

Comment:
```suggestion
// NOTE: we should probably move this routine to the synthetic shadow folder
```

Reply:
This entire comment is outdated, now that we are using this code path for light DOM slotting. Let me get rid of it.